mandos/src/account.rs

```diff
@@ -61,12 +72,43 @@ impl CheckStorage {
 	}
 }
 
+#[derive(Debug)]
+pub enum CheckEsdt {
```

Comment:
You can avoid having to use an extra Option, by having DefaultStar, same as with CheckValue. DefaultStar is identical to Star in behaviour, but it occurs when nothing is written in the json. They are separate just so we can reconstruct the mandos the other way around, no other function. DefaultStar would be the Option::None.